README.md

```diff
@@ -163,11 +163,30 @@ jobs:
       - uses: erlef/setup-beam@v1
         with:
           otp-version: '24'
-          gleam-version: '0.16'
-          rebar3-version: '3.16'
-      - run: rebar3 ct
+          gleam-version: '0.19.0-rc3'
+      - run: gleam test
+```
+
+### Example (Gleam on Ubuntu without OTP)
+
+```yaml
+# create this in .github/workflows/ci.yml
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: false
+          gleam-version: '0.19.0-rc3'
+      - run: gleam check
 ```
 
+**Note**: The `otp-version: false` config is only applicable when installing Gleam.
```

Comment:
```suggestion
**Note**: the `otp-version: false` input is only applicable when installing Gleam.
```

Reply:
Regarding the small caps, I'll be fixing the rest of the README in a different pull request.